command/harness/dlite/dlite.go

```diff
@@ -48,6 +48,16 @@ func RegisterDlite(app *kingpin.Application) {
 		StringVar(&c.poolFile)
 }
 
+// Iterate over the list of pools and register the tags for the pools it supports
+func parseTags(pf *config.PoolFile) []string {
+	tags := []string{}
+	for i := range pf.Instances {
+		t := pf.Instances[i].Platform.OS + "-" + pf.Instances[i].Platform.Arch
```

Comment:
os is used as darwin instead of macos

Reply:
let's keep it for now. We can check with brad whether darwin is a better alternative than mac and update accordingly

Reply:
we can review it when mac os support is added back